File: typescript/src/models/delegatePaymentResponse.ts
// this file is @generated

export interface DelegatePaymentResponse {
  id: string;
  created: string;
  metadata: { [key: string]: string };
}

export const DelegatePaymentResponseSerializer = {
  _fromJsonObject(object: any): DelegatePaymentResponse {
    return {
      id: object["id"],
      created: object["created"],
      metadata: object["metadata"],
    };
  },

  _toJsonObject(self: DelegatePaymentResponse): any {
    return {
      id: self.id,
      created: self.created,
      metadata: self.metadata,
    };
  },
};

File: rust/src/models/delegate_payment_response.rs
// this file is @generated
use serde::{Deserialize, Serialize};

#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
pub struct DelegatePaymentResponse {
    pub id: String,
    pub created: String,
    pub metadata: std::collections::HashMap<String, String>,
}
